specs/engage-digital_openapi3.yaml

```diff
@@ -7978,6 +8031,99 @@ components:
           items:
             type: string
           type: array
+    Survey:
+      properties:
+        id:
+          type: string
+        created_at:
+          format: date-time
+          type: string
+        updated_at:
+          format: date-time
+          type: string
+        name:
+          description: |-
+            Name displayed in ED admin interface.
+            Is not displayed to end users.
+          type: string
+        link:
+          description: |-
+            Link to the survey on the provider website.
+          type: string
+        active:
+          type: boolean
+        from_name:
+          description: |-
+            Name displayed as sender when survey is sent to end user.
+            Currently only applies to surveys sent by email.
+          type: string
+        instant_send_threshold:
+          description: |-
+            All intervention whose duration is shorter than this value (in seconds) will have the survey sent instantly.
+          type: number
+        max_time_since_last_reply:
+          description: |-
+            If last message from end user is older than this value (in seconds), no survey will be sent.

```

Comment:
```suggestion
            If last message from end user is older than this period (in seconds), no survey will be sent.
```
or
```suggestion
            If last message from end user is older than this duration (in seconds), no survey will be sent.
```
WDYT?

Reply:
Ah nice duration is more understandable 👍

Reply:
Done in e4f1502